app/src/server.js

```diff
@@ -59,7 +64,17 @@ app.get(
   }
 );
 
-const server = app.listen(port, function() {
+// Setup and start the HTTPS server.
+
+const key = fs.readFileSync("./certs/key.pem", "utf-8");
```

Comment:
these paths should be configurable via ENV vars

Reply:
Those files are mounted in the Docker container always in the same location, I don't think we need an ENV here

Reply:
it's always convenient to being able to override a conf via env vars (see [12 factor apps](https://12factor.net/config)), for instance if you run the script locally, it's not big deal, you can just add:

``` js
const cert_key_path = process.env.CERT_KEY_PATH || "./certs/key.pem"; 
```